### PR TITLE
Website fixes

### PR DESCRIPTION
**Pull request by drammock**
- some refinements to the colors, making the light-mode more similar to current stable and making the dark-mode visually similar to the light-mode (in terms of things like heading prominence, difference between link color and heading color, etc)
- removes duplicate search icon (the theme now puts it in automatically, which is probably a bug)
- fixes the dark/light logo jitter by using SVG logos for BIDS

addresses (but does not close) #10977 for the 1.1 release

### CODE REVIEW

File: tutorials/epochs/20_visualize_epochs.py [hunk 1]
```diff
@@ -42,7 +42,8 @@
 # Plotting ``Epochs`` as time series
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Interactivity in pipelines and scripts
+# .. note:: Interactivity in pipelines and scripts
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/epochs/20_visualize_epochs.py:46]:
```suggestion
# .. admonition:: Interactivity in pipelines and scripts
#     :class: sidebar note
```

File: tutorials/epochs/30_epochs_metadata.py [hunk 2]
```diff
@@ -32,7 +32,8 @@
 # Viewing ``Epochs`` metadata
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Restrictions on metadata DataFrames
+# .. note:: Restrictions on metadata DataFrames
+#    :class: sidebar
```
Comment (drammock) [inline tutorials/epochs/30_epochs_metadata.py:36]:
```suggestion
# .. admonition:: Restrictions on metadata DataFrames
#    :class: sidebar note
```

File: tutorials/evoked/10_evoked_overview.py [hunk 3]
```diff
@@ -83,7 +83,8 @@
 # Subsetting ``Evoked`` data
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Evokeds are not memory-mapped
+# .. note:: Evokeds are not memory-mapped
+#   :class: sidebar
```
Comment (drammock) [inline tutorials/evoked/10_evoked_overview.py:87]:
```suggestion
# .. admonition:: Evokeds are not memory-mapped
#   :class: sidebar note
```

File: tutorials/evoked/20_visualize_evoked.py [hunk 4]
```diff
@@ -55,7 +55,8 @@
 # Plotting signal traces
 # ^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Butterfly plots
+# .. note:: Butterfly plots
+#    :class: sidebar
```
Comment (drammock) [inline tutorials/evoked/20_visualize_evoked.py:59]:
```suggestion
# .. admonition:: Butterfly plots
#    :class: sidebar note
```

File: tutorials/forward/50_background_freesurfer_mne.py [hunk 5]
```diff
@@ -83,7 +83,8 @@
 # location ``(x, y, z)`` in the *scanner's native coordinate frame* is saved in
 # the image's *affine transformation*.
 #
-# .. sidebar:: Under the hood
+# .. note:: Under the hood
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/forward/50_background_freesurfer_mne.py:87]:
```suggestion
# .. admonition:: Under the hood
#     :class: sidebar note
```

File: tutorials/forward/50_background_freesurfer_mne.py [hunk 6]
```diff
@@ -249,7 +250,8 @@ def imshow_mri(data, img, vox, xyz, suptitle):
 print(nasion_head)  # note it's in "head" coordinates
 
 # %%
-# .. sidebar:: Head coordinate frame
+# .. note:: Head coordinate frame
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/forward/50_background_freesurfer_mne.py:254]:
```suggestion
# .. admonition:: Head coordinate frame
#     :class: sidebar note
```

File: tutorials/intro/20_events_from_raw.py [hunk 7]
```diff
@@ -121,7 +121,8 @@
 print(events[:5])  # show the first 5
 
 # %%
-# .. sidebar:: The middle column of the Events array
+# .. note:: The middle column of the Events array
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/intro/20_events_from_raw.py:125]:
```suggestion
# .. admonition:: The middle column of the Events array
#     :class: sidebar note
```

File: tutorials/intro/40_sensor_locations.py [hunk 8]
```diff
@@ -47,7 +47,8 @@
 #
 # Working with built-in montages
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-# .. sidebar:: Computing sensor locations
+# .. note:: Computing sensor locations
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/intro/40_sensor_locations.py:51]:
```suggestion
# .. admonition:: Computing sensor locations
#     :class: sidebar note
```

File: tutorials/intro/40_sensor_locations.py [hunk 9]
```diff
@@ -124,7 +125,8 @@
 # Plotting 2D sensor locations like EEGLAB
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: The ``sphere`` keyword is available in many places!
+# .. hint:: The ``sphere`` keyword is available in many places!
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/intro/40_sensor_locations.py:129]:
```suggestion
# .. admonition:: The ``sphere`` keyword is available in many places!
#      :class: sidebar hint
```

File: tutorials/io/10_reading_meg_data.py [hunk 10]
```diff
@@ -143,7 +143,8 @@
 MNE-Python includes the :func:`mne.io.read_raw_kit` and
 :func:`mne.read_epochs_kit` to read and convert Ricoh/KIT MEG data.
 
-.. sidebar:: Channel naming
+.. warning:: Channel naming
+    :class: sidebar
```
Comment (drammock) [inline tutorials/io/10_reading_meg_data.py:147]:
```suggestion
.. admonition:: Channel naming
    :class: sidebar warning
```

File: tutorials/preprocessing/15_handling_bad_channels.py [hunk 11]
```diff
@@ -86,7 +86,8 @@
 raw.info['bads'] = original_bads     # change the whole list at once
 
 # %%
-# .. sidebar:: Blocking execution
+# .. hint:: Blocking execution
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/preprocessing/15_handling_bad_channels.py:90]:
```suggestion
# .. admonition:: Blocking execution
#     :class: sidebar hint
```

File: tutorials/preprocessing/40_artifact_correction_ica.py [hunk 12]
```diff
@@ -67,7 +67,8 @@
 # ICA in MNE-Python
 # ~~~~~~~~~~~~~~~~~
 #
-# .. sidebar:: ICA and dimensionality reduction
+# .. hint:: ICA and dimensionality reduction
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/preprocessing/40_artifact_correction_ica.py:71]:
```suggestion
# .. admonition:: ICA and dimensionality reduction
#     :class: sidebar hint
```

File: tutorials/preprocessing/40_artifact_correction_ica.py [hunk 13]
```diff
@@ -215,7 +216,8 @@
 # Fitting and plotting the ICA solution
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
-# .. sidebar:: Ignoring the time domain
+# .. hint:: Ignoring the time domain
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/preprocessing/40_artifact_correction_ica.py:220]:
```suggestion
# .. admonition:: Ignoring the time domain
#     :class: sidebar hint
```

File: tutorials/preprocessing/45_projectors_background.py [hunk 14]
```diff
@@ -203,7 +203,8 @@ def setup_3d_axes():
 # :math:`y`, and :math:`z` directions, the set of projected points have only
 # two *effective* dimensions (i.e., they are constrained to a plane).
 #
-# .. sidebar:: Terminology
+# .. note:: Terminology
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/preprocessing/45_projectors_background.py:207]:
```suggestion
# .. admonition:: Terminology
#     :class: sidebar note
```

File: tutorials/preprocessing/50_artifact_correction_ssp.py [hunk 15]
```diff
@@ -462,7 +462,8 @@
 #
 # Visualizing SSP sensor-space bias via signal reconstruction
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-# .. sidebar:: SSP reconstruction
+# .. note:: SSP reconstruction
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/preprocessing/50_artifact_correction_ssp.py:466]:
```suggestion
# .. admonition:: SSP reconstruction
#     :class: sidebar note
```

File: tutorials/raw/10_raw_overview.py [hunk 16]
```diff
@@ -28,7 +28,8 @@
 # Loading continuous data
 # ^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Datasets in MNE-Python
+# .. note:: Datasets in MNE-Python
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/10_raw_overview.py:32]:
```suggestion
# .. admonition:: Datasets in MNE-Python
#     :class: sidebar note
```

File: tutorials/raw/10_raw_overview.py [hunk 17]
```diff
@@ -95,7 +96,8 @@
 # Querying the Raw object
 # ^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Attributes vs. Methods
+# .. note:: Attributes vs. Methods
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/10_raw_overview.py:100]:
```suggestion
# .. admonition:: Attributes vs. Methods
#     :class: sidebar hint
```

File: tutorials/raw/10_raw_overview.py [hunk 18]
```diff
@@ -164,7 +166,8 @@
 # Time, sample number, and sample index
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
-# .. sidebar:: Sample numbering in VectorView data
+# .. warning:: Sample numbering in VectorView data
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/10_raw_overview.py:170]:
```suggestion
# .. admonition:: Sample numbering in VectorView data
#     :class: sidebar warning
```

File: tutorials/raw/10_raw_overview.py [hunk 19]
```diff
@@ -188,7 +191,8 @@
 # Modifying ``Raw`` objects
 # ^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: ``len(raw)``
+# .. warning:: ``len(raw)``
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/10_raw_overview.py:195]:
```suggestion
# .. admonition:: ``len(raw)``
#     :class: sidebar warning
```

File: tutorials/raw/10_raw_overview.py [hunk 20]
```diff
@@ -251,7 +255,8 @@
 # Changing channel name and type
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
-# .. sidebar:: Long channel names
+# .. note:: Long channel names
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/10_raw_overview.py:259]:
```suggestion
# .. admonition:: Long channel names
#     :class: sidebar note
```

File: tutorials/raw/20_event_arrays.py [hunk 21]
```diff
@@ -76,7 +76,8 @@
 mne.find_events(raw, stim_channel='STI 014')
 
 # %%
-# .. sidebar:: Including/excluding events
+# .. hint:: Including/excluding events
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/raw/20_event_arrays.py:80]:
```suggestion
# .. admonition:: Including/excluding events
#     :class: sidebar hint
```

File: tutorials/simulation/10_array_objs.py [hunk 22]
```diff
@@ -14,15 +14,16 @@
...
 
+import mne
 
 # %%
 # Creating `~mne.Info` objects
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Info objects
+# .. note:: Info objects
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/simulation/10_array_objs.py:26]:
```suggestion
# .. admonition:: Info objects
#     :class: sidebar note
```

File: tutorials/simulation/10_array_objs.py [hunk 23]
```diff
@@ -70,7 +71,8 @@
 info.set_montage('standard_1020')
 
 # %%
-# .. sidebar:: Info consistency
+# .. warning:: Info consistency
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/simulation/10_array_objs.py:75]:
```suggestion
# .. admonition:: Info consistency
#     :class: sidebar warning
```

File: tutorials/simulation/10_array_objs.py [hunk 24]
```diff
@@ -95,7 +97,8 @@
 # Creating `~mne.io.Raw` objects
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# .. sidebar:: Units
+# .. note:: Units
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/simulation/10_array_objs.py:101]:
```suggestion
# .. admonition:: Units
#     :class: sidebar note
```

File: tutorials/stats-sensor-space/10_background_stats.py [hunk 25]
```diff
@@ -395,7 +395,8 @@ def plot_t_p(t, p, title, mcc, axes=None):
 # multidimensional (e.g., spatio-temporal) -- because the null distribution
 # will be derived from data in a way that preserves these correlations.
 #
-# .. sidebar:: Effect size
+# .. note:: Effect size
+#     :class: sidebar
```
Comment (drammock) [inline tutorials/stats-sensor-space/10_background_stats.py:399]:
```suggestion
# .. admonition:: Effect size
#     :class: sidebar note
```

**Comment by drammock** [comment]
@larsoner any idea why the actions have been stalled for 2 days on this PR when I requested they be skipped? Could it be some weird interaction with the branch protections / auto merge?

**Comment by larsoner** [comment]
Yeah they sit as "expected". You can override and merge though

**Pull request merged**